cmake/make_dist/MakeDistCheck.cmake

```
@@ -22,6 +22,11 @@ function (RUN_DIST_CHECK PACKAGE_PREFIX EXT)
       ERROR_MSG "Command to untar ${tarball} failed."
    )
 
+   # Ensure a VERSION_CURRENT file is present.
+   if(NOT EXISTS ${PACKAGE_PREFIX}/build/VERSION_CURRENT)
+      message (FATAL_ERROR "Expected tarball to include `build/VERSION_CURRENT` file but did not.")
```

```suggestion
      message (FATAL_ERROR "Expected tarball to contain a `build/VERSION_CURRENT` file, but it does not")
```
Wording tweak suggestion.